README.md

```diff
@@ -1,2 +1,82 @@
-# spacenet
-Testnet for new generation of Filecoin and Interplanetary Consensus (IPC)
+![](./assets/spacenet-header.png)
+
+# Spacenet
+> A new generation Filecoin testnet.
+>
+> Made with ❤ by [ConsensusLab](https://consensuslab.world/)
+
+- Spacenet Faucet: https://consensus-shipyard.github.io/spacenet-faucet/
+- [Spacenet Genesis](./assets/genesis/spacenet.car)
+- Spacenet Bootstraps:
+  - `xxxx`
+  - `xxxx`
+
+## Why Spacenet?
+Spacenet is not _yet another_ Filecoin testnet. Its consensus layer has been modified to integrate [Mir](https://github.com/filecoin-project/mir), a distributed protocol implementation framework. The current version of Spacenet runs an implementation of the Trantor BFT consensus over Mir. 
+_And what does this mean?_ Well, by implementing a high-performant consensus we have increased the throughput of the network significantly while reducing the 30 second block time consensus to around 1 second. As you may already be aware, Filecoin recently launched its [Filecoin Virtual Machine](https://fvm.filecoin.io/) and it will soon release support for user-defined actor. This will on board a great gamut of new applications to the Filecoin network significantly increasing its load. Certain use cases may realize that they need more from Filecoin in terms of throughput and finality times, and here is were Spacenet comes in.
+
+With Spacenet we want to provide developers with a testbed to deploy their FVM use cases and innovate with new Web3 applications. Many of you may be wondering, _but why would I want to develop my application over a high-throughput Filecoin network if my goal is to deploy it on mainnet?_ Well, Spacenet is just the first step towards the deployment of the [InterPlanetary Consensus (IPC)](https://github.com/filecoin-project/FIPs/discussions/419), and our ambitious plan of scaling Filecoin (in terms of performance and new capabilities). Now Spacenet is just Filecoin with a faster consensus algorithm, but in the future it will be much more. Spacenet is just your portal to glimpse what is yet to come to the Filecoin ecosystem (but we are getting ahead of ourselves). 
+
+Spacenet is not only a testbed for developer to test the new improvements to the protocol we are working on, it is also a way for us to test our consensus innovations with real applications and real users. For instance, once IPC is released in Spacenet, developers will be able to deploy their own subnets from Spacenet while maintaining the ability to seamlessly interact with state and applications in the original network from which they have become independent. With this version of Spacenet, we want to test Mir-Trantor, the first consensus algorithm supported by IPC subnets, before we release full support for IPC. 
```

Comment:
```suggestion
Spacenet is not only a developer sandbox to experiment with new protocol improvements, but also a way for us to test our consensus innovations with real applications and real users. Once IPC is released in Spacenet, developers will be able to deploy their own subnets from Spacenet while maintaining the ability to seamlessly interact with state and applications in the original network, from which they have otherwise become independent. With this version of Spacenet, we want to test Mir-Trantor, the first consensus algorithm supported by IPC subnets, before we unleash full IPC support. 
```